src/Header/SetCookie.php

```diff
@@ -81,14 +81,19 @@ class SetCookie implements MultipleHeaderInterface
     protected $quoteFieldValue = false;
 
     /**
-     * @var bool|null
+     * @var null|bool
      */
     protected $httponly;
 
+    /**
+     * @var string
+     */
+    public $type = 'Cookie';
```

Comment:
This should probably not be added: it will change how the object behaves on a public API (serialization like `json_encode()`, mostly). What's the reason for this change?